packages/block-library/src/site-logo/edit.js

```diff
@@ -480,21 +482,25 @@ export default function LogoEdit( {
 
 		if ( ! media.id && media.url ) {
 			// This is a temporary blob image.
+			setTemporaryURL( media?.url );
```

Comment:
```suggestion
			setTemporaryURL( media.url );
```

Nit: There's no need for optional chaining here. See the condition above.